### PR TITLE
Document hangup

### PR DESCRIPTION
**Pull request by SuaYoo**
Documents hangup method

## 📝 To Do

- [ ] All linters pass
- [ ] All tests pass
- [ ] Change documentation based on my changes

## ✋ Manual testing

1. Start Storybook: `cd packages/js/examples/react-audio && npm start`
2. Go to http://localhost:6006/?path=/story/webdialer--example and fill in your username, password etc.
3. Make a call and hangup. Verify call ends

### Desktop

- [ ] Edge (latest)
- [x] Chrome
- [x] Firefox
- [ ] Safari

### CODE REVIEW

File: packages/js/src/Modules/Verto/webrtc/BaseCall.ts [hunk 1]
```diff
@@ -202,6 +202,18 @@ export default abstract class BaseCall implements IWebRTCCall {
     }
   }
 
+  /**
+   * Hangs up the call.
+   *
+   * @param params _For internal use_ Specify custom hangup cause and call ID
```
Comment (SuaYoo) [inline packages/js/src/Modules/Verto/webrtc/BaseCall.ts:208]: It doesn't appear you can hide a parameter or mark it for internal use in Typedoc, so I've added a note in the description.

File: packages/js/src/Modules/Verto/webrtc/BaseCall.ts [hunk 2]
```diff
@@ -202,7 +202,29 @@ export default abstract class BaseCall implements IWebRTCCall {
...
+   */
+  hangup(): void;
+  /**
+   * @internal
+   */
+  hangup(hangupParams, hangupExecute): void;
+  /**
+   * @internal
+   * @param params _For internal use_ Specify custom hangup cause and call ID
+   * @param execute _For internal use_ Allow or prevent execution of `Bye`
```
Comment (SuaYoo) [inline packages/js/src/Modules/Verto/webrtc/BaseCall.ts:222]: There doesn't appear to be a way to omit specific params with Typedoc, even with Typescript function overloading. The closest I could get to an decent render is by specifying `@internal` and rendering it along with the first `hangup()` typing--using `@ignore` or `@hidden` hides `hangup` altogether.

This is how it looks rendered:
Screen Shot 2020-12-04 at 1 48 56 PM

This way we could omit `@internal` methods when rendering in Developer Docs or other custom templates; any other ideas?

File: packages/js/src/Modules/Verto/webrtc/BaseCall.ts [hunk 3]
```diff
@@ -202,7 +202,29 @@ export default abstract class BaseCall implements IWebRTCCall {
...
+   */
+  hangup(): void;
+  /**
+   * @internal
+   */
+  hangup(hangupParams, hangupExecute): void;
+  /**
+   * @internal
+   * @param hangupParams _For internal use_ Specify custom hangup cause and call ID
+   * @param hangupExecute _For internal use_ Allow or prevent execution of `Bye`
```
Comment (SuaYoo) [inline packages/js/src/Modules/Verto/webrtc/BaseCall.ts:222]: There doesn't appear to be a way to omit specific params with Typedoc, even with Typescript function overloading. The closest I could get to an decent render is by specifying `@internal` and rendering it along with the first `hangup()` typing--using `@ignore` or `@hidden` hides `hangup` altogether.

This is how it looks rendered:
Screen Shot 2020-12-04 at 1 53 19 PM

This way we could omit `@internal` methods when rendering in Developer Docs or other custom templates; any other ideas?

**Review by DeividVeloso — APPROVED** [review]

File: packages/js/src/Modules/Verto/webrtc/BaseCall.ts (hunk 3 above)
Comment (DeividVeloso) [reply packages/js/src/Modules/Verto/webrtc/BaseCall.ts:222]: I don't know.

Any thoughts @hugobdev?

**Pull request merged**